### PR TITLE
Fix end call bugs

### PR DESCRIPTION
**Pull request by PorterNan**
# What

Before:
image

image

After:
image
image
image

https://skype.visualstudio.com/SPOOL/_workitems/edit/3666366
https://skype.visualstudio.com/SPOOL/_workitems/edit/3666365
# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/EndCallButton.tsx [hunk 1]
```diff
@@ -208,3 +218,7 @@ const lightThemeCallButtonStyles = {
     color: lightTheme.callingPalette.iconWhite
   }
 };
+
+const menupButtonPadding = {
+  root: { padding: '0 12px' }
```
Comment (mgamis-msft) [inline packages/react-components/src/components/EndCallButton.tsx:223]: nit: let's use `0 0.75rem` for padding here :)

File: packages/react-composites/src/composites/CallComposite/components/Prompt.tsx [hunk 2]
```diff
@@ -27,16 +28,18 @@ export interface PromptProps {
...
   onCancel?: () => void;
+  styles?: Partial<IModalStyles>;
 }
 
 /**
  * @private
  */
 export const Prompt = (props: PromptProps): JSX.Element => {
   const theme = useTheme();
+  const styles = props.styles ?? modalStyles;
```
Comment (mgamis-msft) [inline packages/react-composites/src/composites/CallComposite/components/Prompt.tsx:39]: nit: I think we should change this to `const styles = concatStyles(modalStyles, props.styles)` just so not all default css properties are overridden when you assign styles. But you can still override all default styles.

File: packages/react-composites/src/composites/CallComposite/components/buttons/EndCall.tsx [hunk 3]
```diff
@@ -28,7 +28,30 @@ export const EndCall = (props: {
...
+    }),
+    [localeStrings.call.endCallConfirmButtonLabel, localeStrings.call.leaveConfirmDialogTitle]
+  );
+
+  /* @conditional-compile-remove(end-call-options) */
+  const leaveDiaglogLabels = useMemo(
+    () => ({
+      confirmButtonLabel: localeStrings.call.leaveConfirmButtonLabel,
+      heading: localeStrings.call.leaveConfirmDialogTitle,
+      text: localeStrings.call.leaveConfirmDialogTitle
```
Comment (mgamis-msft) [inline packages/react-composites/src/composites/CallComposite/components/buttons/EndCall.tsx:45]: Did we want the heading and text to be the same?

File: packages/react-composites/src/composites/CallComposite/components/buttons/EndCall.tsx [hunk 4]
```diff
@@ -28,7 +28,30 @@ export const EndCall = (props: {
   const localeStrings = useLocale().strings;
 
   /* @conditional-compile-remove(end-call-options) */
-  const { onHangUp, enableEndCallMenu } = hangUpButtonProps;
+  const endCallDiaglogLabels = useMemo(
+    () => ({
+      confirmButtonLabel: localeStrings.call.endCallConfirmButtonLabel,
+      heading: localeStrings.call.leaveConfirmDialogTitle,
```
Comment (dmceachernmsft) [inline packages/react-composites/src/composites/CallComposite/components/buttons/EndCall.tsx:34]: This should use different strings than leaveDialogLabels I think?

Comment (PorterNan) [reply packages/react-composites/src/composites/CallComposite/components/buttons/EndCall.tsx]: Nice catch! Fixed!

**Review by dmceachernmsft — APPROVED** [review]

**Pull request merged**